site/schemas/create-consequence-stops.schema.ts

```diff
@@ -0,0 +1,130 @@
+import { Severity } from "@create-disruptions-data/shared-ts/enums";
+import { z } from "zod";
+import { setZodDefaultError, zodTimeInMinutes } from "../utils";
+
+export const stopsImpactedSchema = z.object({
+    id: z.number({
+        required_error: "Id is required",
+        invalid_type_error: "Id must be a number",
+    }),
+    atcoCode: z.string({
+        required_error: "Atco code is required",
+        invalid_type_error: "Atco code must be a string",
+    }),
+    naptanCode: z
+        .string({
+            required_error: "Naptan code is required",
+            invalid_type_error: "Naptan code must be a string",
+        })
+        .optional(),
+    commonName: z.string({
+        required_error: "Common name is required",
+        invalid_type_error: "Common name must be a string",
+    }),
+    street: z
+        .string({
+            required_error: "Street is required",
+            invalid_type_error: "Street must be a string",
+        })
+        .optional(),
+    indicator: z.string().optional(),
+    bearing: z
+        .string({
+            required_error: "Bearing is required",
+            invalid_type_error: "Bearing must be a string",
+        })
+        .optional(),
+    nptgLocalityCode: z
+        .string({
+            required_error: "Nptg locality code is required",
+            invalid_type_error: "Nptg locality code must be a string",
+        })
+        .optional(),
+    localityName: z
+        .string({
+            required_error: "Locality name is required",
+            invalid_type_error: "Locality name must be a string",
+        })
+        .optional(),
+    parentLocalityName: z
```

Comment:
All of these error messages need to be changed if they reference a 'string' - its not a term users will know.